setup.py

```diff
@@ -26,37 +14,36 @@
 import distro
 
 extras= {
-	'core': []
-	}
+    'core': []
+}
 
 #NOTE Package data is inside MANIFEST.In
 
 setuptools.setup(
-        name='diambraArena',
-        url='https://github.com/diambra/diambraArena',
-        version='1.0',
-        author="DIAMBRA Team",
-        author_email="info@diambra.ai",
-        description="DIAMBRA™ Arena. Built with OpenAI Gym Python interface, easy to use,\ntransforms popular video games into Reinforcement Learning environments",
-        long_description="DIAMBRA™ Arena. Built with OpenAI Gym Python interface, easy to use,\ntransforms popular video games into Reinforcement Learning environments",
-        long_description_content_type="Reinforcement Learning",
-        license='GNU Affero GPL',
-        install_requires=[
-            'pip>=21',
-            'setuptools',
-            'distro>=1',
-            'gym>=0.17.1',
-            'jupyter>=1.0.0',
-            'testresources',
-            'inputs',
-            'screeninfo',
-            'tk',
-            'opencv-contrib-python>=4.4.0.42',
-            'opencv-python>=4.4.0.42',
-            'grpcio',
-            'grpcio-tools'],
-        #packages=['diambraArena','diambraArena/wrappers','diambraArena/utils'],
-        packages=[package for package in find_packages() if package.startswith("diambraArena")],
-        include_package_data=True,
-        extras_require=extras
-        )
+    name='diambraArena',
+    url='https://github.com/diambra/diambraArena',
+    version='2.0a0',
+    author="DIAMBRA Team",
+    author_email="info@diambra.ai",
+    description="DIAMBRA™ Arena. Built with OpenAI Gym Python interface, easy to use,\ntransforms popular video games into Reinforcement Learning environments",
```

Comment:
Let's change this to "DIAMBRA™ Arena. Built with OpenAI Gym, easy to use, transforms popular video games into Reinforcement Learning environments." removing the \n new line